package.json

```diff
@@ -38,67 +38,77 @@
-    "unorm": "1.6.0"
+    "@react-native-community/async-storage": "^1.12.0",
+    "@react-native-community/cli": "^4.13.0",
+    "@react-native-community/masked-view": "^0.1.10",
+    "@react-native-community/status-bar": "^1.0.3",
+    "@react-native-firebase/app": "^8.4.3",
+    "@react-native-firebase/iid": "^7.4.4",
+    "@react-native-firebase/messaging": "^7.8.6",
+    "@react-navigation/drawer": "^5.9.1",
+    "@sentry/react-native": "^1.8.2",
+    "i18next": "^19.7.0",
+    "jetifier": "^1.6.6",
+    "moment": "^2.29.0",
+    "prop-types": "^15.7.2",
+    "react": "16.13.1",
+    "react-i18next": "^11.7.3",
+    "react-native": "0.63.2",
+    "react-native-cli": "^2.0.1",
+    "react-native-device-info": "^6.0.4",
+    "react-native-dotenv": "^2.4.0",
+    "react-native-fs": "^2.16.6",
+    "react-native-gesture-handler": "^1.8.0",
+    "react-native-image-crop-picker": "^0.35.0",
+    "react-native-image-zoom-viewer": "^3.0.1",
+    "react-native-linear-gradient": "^2.5.6",
+    "react-native-localize": "^1.4.1",
+    "react-native-reanimated": "^1.13.1",
+    "react-native-render-html": "^4.2.4",
+    "react-native-safe-area-context": "^3.1.8",
+    "react-native-screens": "^2.11.0",
+    "react-native-share": "^3.8.3",
+    "react-native-snackbar": "^2.2.3",
+    "react-native-vector-icons": "^7.1.0",
+    "react-native-webview": "^10.9.0",
+    "react-navigation": "^4.4.1",
+    "react-navigation-drawer": "^2.5.1",
+    "react-navigation-stack": "^2.8.3",
+    "react-redux": "^7.2.1",
+    "redux": "^4.0.5",
+    "redux-saga": "^1.1.3",
+    "unorm": "^1.6.0"
   },
   "devDependencies": {
-    "@babel/plugin-proposal-class-properties": "7.5.5",
-    "@babel/plugin-proposal-optional-chaining": "7.2.0",
-    "@babel/plugin-transform-flow-strip-types": "7.4.4",
-    "@babel/plugin-transform-runtime": "7.5.5",
-    "@babel/preset-env": "7.5.5",
-    "@babel/preset-react": "7.0.0",
-    "babel-core": "7.0.0-bridge.0",
-    "babel-eslint": "10.0.3",
-    "babel-jest": "24.9.0",
-    "eslint": "6.0.1",
-    "eslint-config-airbnb": "17.1.1",
-    "eslint-plugin-import": "2.18.2",
-    "eslint-plugin-jsx-a11y": "6.2.3",
-    "eslint-plugin-react": "7.14.3",
-    "eslint-plugin-react-native": "3.7.0",
-    "i18next-scanner": "2.10.2",
-    "jest": "24.9.0",
-    "metro-react-native-babel-preset": "0.55.0",
-    "metro-react-native-babel-transformer": "0.55.0",
-    "react-native-git-upgrade": "0.2.7",
-    "react-test-renderer": "16.8.6",
-    "redux-mock-store": "1.5.3",
-    "redux-saga-test-plan": "3.7.0"
+    "@babel/core": "^7.11.6",
+    "@babel/plugin-proposal-class-properties": "^7.10.4",
+    "@babel/plugin-proposal-optional-chaining": "^7.11.0",
+    "@babel/plugin-transform-flow-strip-types": "^7.10.4",
+    "@babel/plugin-transform-runtime": "^7.11.5",
+    "@babel/preset-env": "^7.11.5",
+    "@babel/preset-react": "^7.10.4",
+    "@babel/runtime": "^7.11.2",
+    "@react-native-community/eslint-config": "^2.0.0",
```

Comment:
Do we still need the airbnb config with this one already installed?

Reply:
I'm not sure, let's also discuss this Monday.